### DISCUSSION TITLE
Dropping connections, wyze app issues, laggy video

### DISCUSSION THREAD

**Discussion by Jabronis18** [mrlt8/docker-wyze-bridge#918] [Q&A]
Amateur here but love the dev that’s been done here. For reference HW is Surface Pro 3 8GB, i5 running latest Ubuntu LTS and running docker wyze-bridge dishing RTSP to Scrypted and then HomeKit from there. 

No matter what config I’ve tried in the yaml, I get consistent dropped connections, video is laggy, and anytime I’m running wyze-bridge, the wyze app is almost unusable (for the Pan V3s I need to at times go into Wyze to pan the cams manually). 

I’ve tried keeping audio disabled, running at HD60 and SD30, I tried @maxfield-allison’s recent config he posted a couple weeks back but no luck getting anything to significantly improve the streams. Any help is appreciated, concerned it could be HW issue on the machine but i would think the SP3 Pro would still have more HP than
 an RP4 despite its age. Thanks in advance! 
 
 
 Current yaml config:

version: '2.4'
services:
    wyze-bridge:
        container_name: wyze-bridge
        restart: unless-stopped
        image: mrlt8/wyze-bridge:latest
        ports:
            - 1935:1935 # RTMP
            - 8554:8554 # RTSP
            - 8888:8888 # HLS
            - 8889:8889 #WebRTC
            - 8189:8189/udp # WebRTC/ICE
            - 5000:5000 # WEB-UI
        environment:
            # [OPTIONAL] (Can be set in the WebUI):
            - WYZE_EMAIL=
            - WYZE_PASSWORD=
            - ENABLE_BOA=true
            - FPS_FIX=true
            - ON_DEMAND=False
            - IGNORE_OFFLINE=true
            - QUALITY=SD30
            - NET_MODE=ANY
            
            #- ROTATE_CAM_BEAR=True
            #- ROTATE_CAM_GAME_ROOM_CAM=True
            # [OPTIONAL] IP Address of the host to enable WebRTC e.g.,:
            - WB_IP=192.168.1.88
            # [OPTIONAL] Wyze API credentials:
            # - API_ID=
            # - API_KEY=

**Comment by maxfield-allison** [comment]
What is your wireless infrastructure like? brand and number of AP's, brief overview of topography?

**Reply by Jabronis18** [reply]
AT&T default router downstairs and one additional Archer A7 ac1750 converted to an AP upstairs all 1Gb speed 


3 Pan v3s
1 Pan v2
1 cam v3
2 Cam v2s

**Reply by maxfield-allison** [reply]
any switches or are you meshing the AP's

**Reply by mrlt8** [reply]
Try adjusting the IGMP settings (Advanced > Network > IPTV/VLAN).  See if **IGMP Snooping** helps.

**Reply by maxfield-allison** [reply]
unfortunately the ATT router is incredibly lacking when it comes to any settings like that. the Archer A7 (from the manual at least) also doesn't have anything you can set. I would bet you're running into wireless multicast flooding from the description of the problem and that just isn't something you can deal with on most consumer routers.

**Reply by maxfield-allison** [reply]
actually, see if this applies 
https://www.tp-link.com/us/configuration-guides/configuring_layer_2_multicast/?configurationId=18230#_idTextAnchor000

**Reply by Jabronis18** [reply]
Been looking up some of the mesh routers, not sure if I’m going d down the right path, but was reading some good things about the wyze mesh routers, not sure if that would make solving the multicast flooding any easier, but recommendations would be helpful.

Not sure if I can afford going the Unifi route but ifs there’s a budget way to do that would be interested - my wife isn’t complaining about the wyze app so it’s more funding my little project of getting all the cameras in the house into HomeKit (we have a couple Vivint cams running through scrypted not giving me hardly any issues comparatively. 

In the meantime I may try bypassing my ATT Router to my Tp-Link AX1500 and take a stab at the link you sent.

**Comment by Jabronis18** [comment]
Only one switch in the garage, and I don’t think I am meshing the APs (extremely noob when it comes to mesh networks)On Jul 15, 2023, at 1:45 PM, Maxfield Allison ***@***.***> wrote:﻿
any switches or are you meshing the AP's

**Comment by Jabronis18** [comment]
Assuming that’s in the router settings but can’t seem to locate that in the
AT&T interface?

On Sat, Jul 15, 2023 at 11:25 PM mrlt8 ***@***.***> wrote:

> Try adjusting the IGMP settings (Advanced > Network > IPTV/VLAN). See if *IGMP
> Snooping* helps.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mrlt8/docker-wyze-bridge/discussions/918#discussioncomment-6458739>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYGEWYTJ35Y77WFDCRGYC4LXQNUJZANCNFSM6AAAAAA2LLGM2I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
-- 
Jeremy LeBaron
SENIOR ACCOUNT EXECUTIVE
P:
(801) 440-7887
E:
***@***.***

Inline image


Inline image


Inline image


Inline image


Inline image
https://mtxb2b.com
3001 Dallas Pkwy
Suite 210
Frisco, TX 75034
Albany, NY | Frisco, TX | Boston, MA | Jaipur, India | Hyderabad, India
Jeremy LeBaron
SENIOR ACCOUNT EXECUTIVE
P:
(801) 440-7887
E:
***@***.***

Inline image


Inline image


Inline image


Inline image


Inline image
https://mtxb2b.com
3001 Dallas Pkwy
Suite 210
Frisco, TX 75034
Albany, NY | Frisco, TX | Boston, MA | Jaipur, India | Hyderabad, India

**Reply by mrlt8** [reply]
Could you try checking the on the Archer A7 ac1750?

**Reply by Jabronis18** [reply]
I just checked, and after not being able to locate the settings in the admin portal, I looked at the article and failed to notice that those settings are only applicable to tplink gigabit switches, name L2 switches, but those settings cant be found on either router :/

**Reply by mrlt8** [reply]
Are there any IPTV options available?

**Reply by Jabronis18** [reply]
So I just switched one of the routers from AP mode to router (which was a bit of a duh moment) but now seeing under advanced settings this:

![ima_6b422e9](https://github.com/mrlt8/docker-wyze-bridge/assets/101469026/14e5fca0-622f-4b51-bf29-38059f5cd08f)

So do I just enable IGMP Snooping, are there any other settings I should be configuring on that page?

**Reply by mrlt8** [reply]
See if those changes help with the connection issue? 

Possibly v3 for the IGMP version. 

BTW, switching from AP to router mode could cause a double NAT on your network is you may need to make some adjustments on your main router/modem.

**Reply by Jabronis18** [reply]
Yep, I will be testing out turning off the WiFi radio on the BGW-320, enabling IP Passthrough and seeing if this works better with the TP-link routers with one in AP mode, better than just the ATT hardware doing everything. Created a different SSID for the TP-link router while testing. 

Do I need to enable IPTV/VLAN and/or do anything with port mode or the IPTV/VLAN Mode (changing from bridge to ISP Profile, etc)?

**Reply by Jabronis18** [reply]
ok so I have set up the network, BGW320 radios are off, the Archer is the primary router with IGMP snooping enabled. I am noticing my wifi speeds are relatively slow, just updated the YAML file to the config maxwell-allison shared, and unfortunately I am not seeing any major improvement. I did try running wyze-bridge with default settings, and still no luck,  not sure what else I can try at this point, aside from new hardware, which is not something I can afford to do in any extensive form at the moment.
